### PR TITLE
RavenDB-14758 NRE in GetMinLastEtag if backup is not created

### PR DESCRIPTION
**Pull request by ayende**

### CODE REVIEW

**Review by grisha-kotler — APPROVED** [review]
LGTM

**Comment by ppekrol** [comment]
Tests are failing because of this change

File: src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs [hunk 1]
```diff
@@ -688,6 +688,11 @@ private long GetMinLastEtag()
                         continue; // if the backup is always full, we don't need to take into account the tombstones, since we never back them up.
 
                     var status = GetBackupStatusFromCluster(_serverStore, context, _database.Name, taskId);
+                    if (status == null)
+                    {
+                        // if there is no status for this, we don't need to take into account tombstones
+                        continue;
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs]:
```suggestion
                        min = 0; // cannot delete the tombstones until we've done a full backup
                        continue;
```

File: src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs [hunk 2]
```diff
@@ -688,6 +688,11 @@ private long GetMinLastEtag()
                         continue; // if the backup is always full, we don't need to take into account the tombstones, since we never back them up.
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs]: This change should also be here

**Comment by ppekrol** [comment]
We are still having issue with 1 test after this change

**Review by grisha-kotler — APPROVED** [review]

**Pull request merged**